app/helpers/admin/url_options_helper.rb

```diff
@@ -22,7 +22,7 @@ def show_url_with_public_and_cachebusted_options(model, url_options = {})
 
   def view_on_website_link_for(model, options = {})
     url_options = options.delete(:url) || {}
-    link_to "View on website", show_url_with_public_and_cachebusted_options(model, url_options), options
+    link_to "View on website (opens in a new tab)", show_url_with_public_and_cachebusted_options(model, url_options), options
```

Comment:
Sorry I  know this was me who said this was fine, but i think we should double check with Nik he is happy for this to be applied across pages.

I think there's a few options if not, we could do something like

```
def view_on_website_link_for(model, options = {})
  url_options = options.delete(:url) || {}
  link_text = options.delete(:link_text) || "View on website"

  link_to link_text  show_url_with_public_and_cachebusted_options(model, 
  url_options), options
end
```

then just pass in the `"View on website (opens in a new tab)"` as an option